### PR TITLE
Provide native binaries for CLI

### LINKED ISSUE #140: [Task] Provide native binaries for bamm-cli for easier usage
**Is your task related to a problem? Please describe.**
For users it is often hard to use the bamm-cli. 
Newest version has to be downloaded. It has to be started with a java installation. It is not always clear which java version is prerequisite, and sometimes a new java version has to be installed. The command line has to be adjusted.

**Describe the solution you'd like**
The bamm-cli should be provided as a native binary for the relevant platforms (windows, mac, linux)

**Describe alternatives you've considered**
Improve documentation of system requirements

**Additional context**
Proposed technical setup: GraalVM

### PR DESCRIPTION
**Pull request by atextor**
Fixes #140.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,9 +49,22 @@ Are you having trouble with SDS SDK? We want to help!
 
 ## Build and contribute
 
-The top level elements of the SDK structure are all carried out as Maven multimodule projects. Building the SDK components of one group (e.g. all core
-components) then is as easy as running `mvn package` or `mvn install`
-in the root directory of the repository.
+The top level elements of the SDK structure are all carried out as Maven multimodule projects.
+Building the SDK requires a Java 17-compatible [GraalVM JDK](https://www.graalvm.org/) with the 'js'
+component installed. If you with to build the bamm-cli (see below), you will also have to have the
```
Comment (RaMisess) [inline README.md:54]: with -> want?

File: core/sds-aspect-model-validator/src/main/java/io/openmanufacturing/sds/aspectmodel/validation/services/AspectModelValidator.java [hunk 2]
```diff
@@ -167,6 +171,9 @@ public List<Violation> validateModel( final Try<VersionedModel> versionedModel )
          // To catch false positives, also try to load the model
          final Try<List<ModelElement>> modelElements = AspectModelLoader.getElements( model );
          if ( modelElements.isFailure() && !(modelElements.getCause() instanceof InvalidRootElementCountException) ) {
+            final ByteArrayOutputStream out = new ByteArrayOutputStream();
```
Comment (RaMisess) [inline core/sds-aspect-model-validator/src/main/java/io/openmanufacturing/sds/aspectmodel/validation/services/AspectModelValidator.java:174]: Will this remain or was it meant only for debugging?

Comment (atextor) [reply core/sds-aspect-model-validator/src/main/java/io/openmanufacturing/sds/aspectmodel/validation/services/AspectModelValidator.java:174]: Yes, this should be removed here.

File: tools/bamm-cli/src/main/java/io/openmanufacturing/sds/aspect/AspectValidateCommand.java [hunk 3]
```diff
@@ -54,8 +61,10 @@ public void run() {
 
       final List<Violation> violations = validator.validateModel( versionedModel );
       if ( details ) {
+         LOG.debug( "Printing detailed validation results" );
```
Comment (RaMisess) [inline tools/bamm-cli/src/main/java/io/openmanufacturing/sds/aspect/AspectValidateCommand.java:64]: Do we really need this? Doesn't the output formats differ enough to easily recognize which version is being printed?

**Review by RaMisess — APPROVED** [review]

File: tools/bamm-cli/src/main/java/io/openmanufacturing/sds/aspect/AspectValidateCommand.java (hunk 3 above)
Comment (atextor) [reply tools/bamm-cli/src/main/java/io/openmanufacturing/sds/aspect/AspectValidateCommand.java:64]: You are right, it's not really necessary.

**Review by michelu89 — APPROVED** [review]

File: .github/workflows/release-workflow.yml [hunk 4]
```diff
@@ -42,75 +39,225 @@ jobs:
...
+        uses: actions/upload-artifact@v3
+        with:
+          name: linux-artifacts
+          path: |
+            nexus-staging/
+            tools/bamm-cli/target/bamm-cli-linux.tar.gz
+            tools/bamm-cli/target/bamm-cli-*.jar
+
+  # The Windows build will build the Windows-specific bamm-cli
+  # and upload the binary binary to the build artifacts
```
Comment (RaMisess) [inline .github/workflows/release-workflow.yml:108]: binary binary

**Review by RaMisess — APPROVED** [review]

**Pull request merged**